tests/stories/test_tier1_coverage.py

```diff
@@ -0,0 +1,227 @@
+import logging
+
+from ocs_ci.ocs import constants
+from ocs_ci.framework.testlib import ManageTest
+from ocs_ci.ocs.cluster import CephCluster
+from tests.helpers import wait_for_resource_state, get_worker_nodes, create_pod
+from ocs_ci.ocs.resources import pod as POD
+
+log = logging.getLogger(__name__)
+
+
+class TestIOMultiplePods(ManageTest):
+    """
+    Run IO on multiple pods in parallel
+    """
+    fio_runtime = 10
+    pvc_size = 5
+    sc_obj_list = []
+    pvc_objs = []
+    pod_objs = []
+    block_pvc = None
+    block_pod = None
+    pvc_changed_to_delete = None
+    pvc_changed_to_retain = None
+
+    def test_story_tier1(
+        self, storageclass_factory, pod_factory, pvc_factory, teardown_factory
+    ):
+        """
+        Covering tier1 functionality in a single test
+        1) Create 4 storage classes for 2 pool types and 2 reclaim policy types
+        2) Create PVCs from all pool types, access modes and reclaim policies
+        3) Create Pod per PVCs and run IO
+        4) Change the reclaim policy of 2 pods (switch)
+        5) Verify IO on the 2 pods
+        6) Create a new pod and attach it to the RWO PVC - it should be pending
+        7) delete the pod that attached to the RWO PVC.  The 2nd pod will run
+            from step 6
+        8) Verify md5sum
+        x) TBD: Delete the 2 pods and verify the reclaim policy change took
+            affect
+        """
+        for interface in [constants.CEPHBLOCKPOOL, constants.CEPHFILESYSTEM]:
+            for reclaim_policy in [
+                constants.RECLAIM_POLICY_RETAIN, constants.RECLAIM_POLICY_DELETE
+            ]:
+
+                log.info(
+                    "Create storage classes for each interface and "
+                    "reclaim policy"
+                )
+                sc_obj = storageclass_factory(
```

Comment:
It can skip that creation for RECLAIM_POLICY_RETAIN, that way it will test both cases.

Reply:
@vasukulkarni , I'm not sure I understand your comment

Reply:
@RazTamir I meant

```python
if reclaim_policy ==  constants.RECLAIM_POLICY_RETAIN:
   # use system provided storage class
    sc_obj = None
else:
    # create a new one
```